pkg/database/openshiftclusters.go

```diff
@@ -23,6 +26,8 @@ const (
 	OpenshiftClustersPrefixQuery        = `SELECT * FROM OpenShiftClusters doc WHERE STARTSWITH(doc.key, @prefix)`
 	OpenshiftClustersClientIdQuery      = `SELECT * FROM OpenShiftClusters doc WHERE doc.clientIdKey = @clientID`
 	OpenshiftClustersResourceGroupQuery = `SELECT * FROM OpenShiftClusters doc WHERE doc.clusterResourceGroupIdKey = @resourceGroupID`
+
+	openShiftClusterContainerName = "OpenShiftClusters"
```

Comment:
if we're pulling this out, we should leverage it in our above queries.

Reply:
yeah point, I will do that.